src/constants/queryHeaders.ts

```diff
@@ -117,8 +117,8 @@ export const queryData: Array<IQueryHeaders> = [
       domain_desc: "Total",
       geographicAreas: ["State", "County"],
       years: {
-        "County": ["1997", "2002", "2007", "2012", "2017"],
-        "State": ["1997", "2002", "2007", "2012", "2017"]
+        "County": ["2022", "2017", "2012", "2007", "2002", "1997"],
+        "State": ["2022", "2017", "2012", "2007", "2002", "1997"]
```

Comment:
In a few cases like this, I re-ordered the year values so these arrays all use the same order: newest to oldest. That doesn't affect any functionality. I just found the consistency helpful.